Tiltfile.dev

```diff
@@ -1,11 +1,11 @@
 local_resource(
-  name = 'dev:lfp-pay-web-build',
+  name = 'dev:penalty-payment-web-build',
   cmd = 'mvn compile',
   deps = ['src']
 )
 
 custom_build(
-   ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/lfp-pay-web',
+   ref = '169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/penalty-payment-web',
```

Comment:
Same as above, might be worth keeping this lfp for now.

Reply:
Revert back to lfp-pay-web as suggested, and keep the eu-west-2